plutus-ledger-api/src/Plutus/V1/Ledger/Api.hs

```diff
@@ -167,40 +151,34 @@ mkTermToEvaluate bs args = do
 -- | Evaluates a script, with a budget that restricts how many resources it can use.
 evaluateScriptRestricting
     :: VerboseMode -- ^ Whether to produce log output
-    -> CostModelParameters -- ^ The cost model to use
     -> ExBudget -- ^ The resource budget which must not be exceeded during evaluation
     -> Script -- ^ The script to evaluate
     -> [Data] -- ^ The arguments to the script
     -> (LogOutput, Either EvaluationError ())
-evaluateScriptRestricting verbose costParams budget p args = swap $ runWriter @LogOutput $ runExceptT $ do
```

Comment:
Perhaps it's for the ledger to decide what cost model to use? We may have several ones between V1 and V2 (because we may need to tweak some parameters after the release), so it probably makes sense to paramaterize evaluator by a `CostModel` instead of hardcoding the default one?

Reply:
OK, but to make it a bit easier to use I've added a comment saying where to find `defaultCostModel` and I've also exported `enormousBudget` from `Cek.ExBudgetMode`.  Maybe the enormous budget could go in `ExBudget.hs` since it's not really CEK-specific.

Reply:
> Perhaps it's for the ledger to decide what cost model to use? 

I believe that's correct. The cost model is an update-able protocol parameter in the ledger, so the genesis key delegates can vote to change it to whatever they want.

Reply:
Maybe that's what `Integer` meant in `CostModelParameters` then? A position in some array of calibration constants of a builtin.